### PR TITLE
= 3.2.11 =

### PR DESCRIPTION
**Pull request by FerhanGit**
* Fix related with trigger emails

= 3.2.10 =
* Segmentation fix related with Group association

= 3.2.9 =
* Fixed displaying of special symbols in trigger emails
* Added checks for existing parameters in hookNewOrder()

= 3.2.8 =
* jQuery is included explicitly now in the module
* Added correct hooks, related to customer edition and properly handling of customer subscription/unsubscription to Mailjet's master contact list

= 3.2.7 =
* Mailjet’s segmentation feature now allows multi-store owners to filter their customers by the store they belong to.

= 3.2.6 =
* Added links to User guide and Support at the plugin footer
* Fixed marketing triggers cron script
* On creation of a new customer, the email and properties are now synced properly to Mailjet.

### CODE REVIEW

File: mailjet.cron.php [hunk 1]
```diff
@@ -295,6 +295,10 @@ function findCode($code)
 					$content = str_replace('{'.$key.'}', $value, $content);
 				}
 
+                $content = str_replace('{shop_name}', $shop_name, $content);
+				$content = str_replace('{shop_url}', $shop_url, $content);
+				$content = str_replace('{shop_logo}', $shop_logo, $content);
+
```
Comment (Quetzacoalt91) [inline mailjet.cron.php]: You can make it quicker to execute:

```php
$tags_search = array('{shop_name}', '{shop_url}', '{shop_logo}');
$tags_replace = array($shop_name, $shop_url, $shop_logo);
$seltemplate = str_replace($tags_search, $tags_replace, $seltemplate);
```

File: mailjet.php [hunk 2]
```diff
@@ -198,19 +198,18 @@ public function install()
...
-			$this->registerHook('actionAdminCustomersControllerDeleteBefore');
-
 		return (parent::install()
 			&& $this->loadConfiguration()
-			&& $this->registerHook('BackOfficeHeader')
+            && $this->registerHook('actionAdminCustomersControllerSaveBefore')
+			&& $this->registerHook('actionAdminCustomersControllerSaveAfter')
+			&& $this->registerHook('actionAdminCustomersControllerStatusAfter')
+			&& $this->registerHook('actionAdminCustomersControllerDeleteBefore')
+			&& $this->registerHook('actionObjectCustomerUpdateAfter')
```
Comment (Quetzacoalt91) [inline mailjet.php]: You may also want to register this new hook in an upgrade file ?

File: mailjet.cron.php (hunk 1 above)
Comment (FerhanGit) [inline mailjet.cron.php]: Hi Thomas :)
This one is fixed and the PR is updated.
The other one 
https://github.com/PrestaShop/mailjet/pull/24/files#r44911737  is OK for 
us - Thank you!

Regards!

Comment (Quetzacoalt91) [inline mailjet.cron.php]: Hi,

Thanks for this new commit.
However I do not understand what you mean for the upgrade. Do you mean you won't add it or you are going to commit it later ?

Comment (FerhanGit) [inline mailjet.cron.php]: Nop, this comment is fixed and the fix is added to the PR

I mention hat is OK for us to not have the fix in upgrade script for the 
https://github.com/PrestaShop/mailjet/pull/24/files#r44911737 comment

If there is something unclear don`t hesitate to contact me again. Have a 
nice day!

Comment (FerhanGit) [inline mailjet.cron.php]: If you wish you can add me to the skype - ferhan_ismailov to make 
communication faster.

regards!

**Comment by FerhanGit** [comment]
Thank you @Thomas!

Wish you have a nice day!

**Comment by Quetzacoalt91** [comment]
Hi @FerhanGit,

Thanks, your PR has just been merged. As I said, please note that the merchants upgrading their module to this new version won't see the hook `actionObjectCustomerUpdateAfter` registered.

The new version will be available from tomorrow.

Best regards

**Pull request merged**